docs/model-management-tutorial.md

```diff
@@ -0,0 +1,108 @@
+---
+id: model-management-tutorial
+title: Manage and Deploy a Model
+description: Manage and Deploy a Model
+---
+
+<div class="label-sect">
+  <div class="ee-only tooltip">Enterprise
+    <span class="tooltiptext">Applicable to Enterprise Edition</span>
+  </div>
+  <div class="deploy-only tooltip">Deploy
+    <span class="tooltiptext">Applicable to Deploy Edition</span>
+  </div>
+</div>
+<br>
+
+In this tutorial, we will use the MNIST model in TensorFlow 2 as an example to show how to train, manage, and deploy a model.
+
+## What we need?
+
+- Remember to follow [configuration](model-configuration) to enable model management in your group, contact your admin if it is not enabled yet.
+- Remember to enable model deployment in your group, contact your admin if it is not enabled yet.
+![](assets/mdeploy_enable.png)
+- The `TensorFlow 2.4` image `infuseai/docker-stacks:tensorflow-notebook-v2-4-1-dbdcead1`.
+- An instance type >= minimal requirement (CPU=1, GPU=0, Mem=2G).
+- The prepared notebook file of the example,
+
+    Download [model_management_tutorial.ipynb](assets/model_management_tutorial.ipynb).
+    This example file is referred to [TensorFlow 2 quickstart for beginners](https://www.tensorflow.org/tutorials/quickstart/beginner) with added cell to enable [MLflow autologging API](https://www.mlflow.org/docs/latest/python_api/mlflow.tensorflow.html#mlflow.tensorflow.autolog).
+    ![](assets/model-mgt-mlflow-tf-autolog.png)
+
+- Choose a group with enabled Shared Volume (a.k.a Group Volume).
+
+>Please have the image, the instance type on PrimeHub, or request administrators for assistance before we start.
+
+## Steps
+
+1. Enter Notebooks from User Portal, select the image, the instance type, and start a notebook.
+
+1. While inside the group volume, copy/drag the downloaded `model_management_tutorial.ipynb` to File Browser and open it.
+
+1. **Run All Cells** in the notebook, we will see a newly completed run under the `Default` experiment in the MLflow UI. Now clicking on this run.
```

Comment:
Add some simple instructions of how to open the MLflow UI?

Reply:
ok, will add